content/blog/backstage-weekly-7-open-governance/index.md

```diff
@@ -0,0 +1,50 @@
+---
+
+Backstage continues to take big strides towards becoming a community controlled CNCF sandbox project. This week, the project joined peers like Helm, Envoy Proxy and Kubernetes in adopting an open governance model.
+
+> The [open governance](https://github.com/opengovernance/opengovernance.dev) of a project determines who has influence and control beyond what is legally required in an open source license.
+
+Practically speaking, this makes it easy to see who maintains Backstage, what their responsibilities are and why they deserve those responsibilities.
+
+Anyone can become a maintainer. To become a maintainer, you need to demonstrate commitment to the project, the ability to write good code and collaborate effectively with the Backstage team. There is also now a clear nomination process for new maintainers.
+To learn more, check out the [GOVERNANCE](https://github.com/backstage/backstage/blob/master/GOVERNANCE.md) file in the backstage repo.
+
+## Thoughtworks Technology Radar
+
+Backstage recently claimed a spot on the Assess ring of the Thoughtworks Technology Radar.
+
+In the Technology Radar Vol. 23, linked below, Cassie Shum, Head of Cloud Partnerships at Thoughtworks, mentioned that many of their clients are trying to standardize the tools and technologies they use and introduce lean governance to their organizations. They believe that increasing consistency can streamline development and reduce the time to market for new services. Cassie sees Backstage as a way to do that because it makes “discoverability a first-class citizen”.
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/CUTSnAutoAM?start=1176" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+## Merged last week...
+
+58 pull requests were merged last week. Here are the highlights.
+
+### SonarCloud plugin
+
+In a great demonstration of collaboration, a SonarCloud/SonarQube plugin was added to Backstage this week by Dominik Henneke of SDA-SE. [#3160](https://github.com/backstage/backstage/pull/3160)
+
+A SonarCloud plugin was originally [requested by Julien Lind](https://github.com/backstage/backstage/issues/3063) at Decathlon on the 23rd of October. SDA-SE responded that they had already built this internally. Then they open sourced it within 7 days for everyone else in the Backstage community to use. The power of open source is right there for everyone to see!
+
+![Code quality metrics in the SonarCloud plugin](./code-quality.png)
+
+### Component validation
+
+The SonarCloud plugin isn’t the only thing that SDA-SE have been working on with respect to Backstage. Oliver Sand added a way to validate component metadata files before adding them into the catalog. [#3013](https://github.com/spotify/backstage/pull/3013) I can imagine this concept being expanded to include validation steps in CI/CD steps in the future.
```

Comment:
Should the link to 3013 be part of the previous sentence instead?

Reply:
@martina-if Fixed. You will need to approve again.